### PR TITLE
Information filter

### PR DESCRIPTION
**Pull request by koaning**

### CODE REVIEW

**Comment by koaning** [comment]
using nb sphinx for docs 

image

File: sklego/preprocessing.py [hunk 1]
```diff
@@ -420,3 +420,68 @@ def transform(self, X):
         X = check_array(X, estimator=self)
 
         return X @ self.inv_R_ / self.normalization_vector_
+
+
+def scaler_(vec, unto):
```
Comment (MBrouns) [inline sklego/preprocessing.py]: You actually already compute the projection here, right? Should this function be called `scalar_projection`? and the `project_` then `vector_projection`?

Comment (koaning) [reply sklego/preprocessing.py]: it'd be a minor rename only, but sure, it's more explicit that way.

File: sklego/preprocessing.py [hunk 2]
```diff
@@ -420,3 +420,68 @@ def transform(self, X):
...
+    return vec.dot(unto)/unto.dot(unto)
+
+
+def project_(vec, unto):
+    return scaler_(vec, unto) * unto
+
+
+class InformationFilter(BaseEstimator, TransformerMixin):
```
[the remark this answers is not in the document]
Comment (koaning) [reply sklego/preprocessing.py]: ah yes.

**Comment by koaning** [comment]
@MBrouns could you send a screenshot? I'm not sure where the "how it works" is supposed to be.

**Comment by koaning** [comment]
image

found and fixed it

File: sklego/preprocessing.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (koaning) [reply sklego/preprocessing.py]: done.

File: sklego/preprocessing.py (hunk 1 above)
Comment (koaning) [reply sklego/preprocessing.py]: done.

**Pull request merged**